syncer/syncer_test.go

```diff
@@ -80,11 +84,26 @@ func (s *testSyncerSuite) SetUpSuite(c *C) {
 
 	s.resetMaster()
 	s.resetBinlogSyncer()
+	s.resetEventsGenerator()
 
 	_, err = s.db.Exec("SET GLOBAL binlog_format = 'ROW';")
 	c.Assert(err, IsNil)
 }
 
+func (s *testSyncerSuite) resetEventsGenerator() {
+	previousGTIDSetStr := "3ccc475b-2343-11e7-be21-6c0b84d59f30:1-14,406a3f61-690d-11e7-87c5-6c92bf46f384:1-94321383"
+	previousGTIDSet, err := gtid.ParserGTID(s.cfg.Flavor, previousGTIDSetStr)
+	if err != nil {
+		log.L().Fatal("", zap.Error(err))
```

Comment:
it's better to use `check.FailNow` to stop unit test

Reply:
Done